src/component/navbar/navbar.jsx

```diff
@@ -1,10 +1,12 @@
+import { useWishlist } from "context/wishlist-context";
 import { useState } from "react";
 import { Link } from "react-router-dom";
 import { useAside } from "../../context/aside-context";
```

Comment:
You can also import everything from context using absolute path

Reply:
done